src/hooks/use-form.jsx

```diff
@@ -13,6 +13,10 @@ export const useForm = ({
   const [errors, setErrors] = useState(initialErrors)
   const [isTouched, setTouched] = useState(getEmptyValues(initialValues, false))
 
+  const isValid = Object.entries(data).every(
+    ([key, value]) => value && !errors[key]
```

Comment:
Did it tested with some "false-like" values?  Do we allow `0` or emty string as `true` value?

Reply:
We don't allow an empty string, but I didn't change the validation rules. For instance, Name or Password can be '0' and there will be no validation errors. Do I need to change the validation as well?

Reply:
I think we can use this validation rule. I think "0"-s in passwords or names have String type, and we should not validate numbers by this rule.